.travis.yml

```diff
@@ -44,6 +44,7 @@ deploy:
     condition: $BUILDTYPE = release && $OMIT_DEPRECATED = false
   edge: true
 
+before_script: chmod +X bundle-test.sh
+
 script:
-  - chmod +X bundle-test.sh
-  - bash bundle-test.sh
+  - ./bundle-test.sh
```

Comment:
Need to source it so you get the environment variables.  Executing it does it in a new shell.

Reply:
and don't make it executable...